### PR TITLE
entoas: integration test for custom oas types

### PR DESCRIPTION
**Pull request by masseelch**

### CODE REVIEW

**Comment by masseelch** [comment]
@a8m This PR needs to wait for contrib to be updated to use latest master of ent.

**Comment by a8m** [comment]
> @a8m This PR needs to wait for contrib to be updated to use latest master of ent. Because the current version generated a bugged version of [runtime.go](https://github.com/ent/contrib/blob/f7712e470ccf358794085e30fe2f376a12c5ce9a/entoas/internal/oastypes/runtime.go#L24)

kk, do you want to create another PR for updating ent? thanks 🙏

File: entoas/internal/oastypes/entc.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+func main() {
+	ex, err := entoas.NewExtension()
+	if err != nil {
+		log.Fatalf("creating entoas extension: %v", err)
+	}
+	err = entc.Generate("./schema", &gen.Config{
+		Templates: []*gen.Template{gen.MustParse(
+			gen.NewTemplate("").Parse(`{{ define "model/stringer" }}// No String() implementation{{ end }}`),
+		)},
```
Comment (a8m) [inline entoas/internal/oastypes/entc.go:36]: I'm afraid users will use it as a reference example (and copy/paste this). Let's remove this template and rename the `string` field in ent/schema. WDYT?

Comment (masseelch) [reply entoas/internal/oastypes/entc.go]: Agreed.

**Review by a8m — APPROVED** [review]
Thanks for adding this @masseelch ❤️

**Pull request merged**